drop-transfer/Cargo.toml

```diff
@@ -19,7 +19,7 @@ tempfile = "3.5.0"
 anyhow = "1.0.70"
 async-trait = { workspace = true }
 drop-analytics = { version = "1.0.0", path = "../drop-analytics" }
-drop-config = { version = "1.0.0", path = "../drop-config" }
+drop-types = { version = "1.0.0", path = "../drop-types" }
```

Comment:
I think it's better to remove `version = "1.0.0"` and let the cargo pick whatever is in the `../drop-types`